src/Components/AdministratorPage/bidAudit/BidAudit.jsx

```diff
@@ -60,16 +81,27 @@ const BidAudit = () => {
       }
       <div className="usa-width-one-whole position-search--results">
         <div className="usa-grid-full position-list">
-          <p className="add-at">
-            <FA name="plus" />
-            {' '}
-            <Link
-              to="#"
-              onClick={onAddClick}
-            >
-              {'Create New Audit Cycle'}
-            </Link>
-          </p>
+          <span className="ba-update-new-wrapper">
+            <span className="ba-update-new">
+              <Tooltip title="Run Dynamic Audit">
+                <FA name="clock-o" />
+                {' '}
```

Comment:
```suggestion
                <FA className="mr-10" name="clock-o" />
```
and a class in _common.scss `.mr-10 { margin-right: 10px; }`

Reply:
the margin right adds too much space between the icon and text & makes it look a little disjointed, i think